### PR TITLE
chore: fix indentation on Game and Root

### PR DESCRIPTION
**Pull request by jmporchet**

### CODE REVIEW

File: imports/ui/Game.jsx [hunk 1]
```diff
@@ -1,34 +1,47 @@
...
+import { Games } from "../api/games.js";
+import { browserHistory } from "react-router";
+import { Meteor } from "meteor/meteor";
+import Timer from "./Timer";
+import PlayerCard from "./PlayerCard";
+import { Tabs, Tab } from "material-ui/Tabs";
 // From https://github.com/oliviertassinari/react-swipeable-views
-import SwipeableViews from 'react-swipeable-views';
-import FontIcon from 'material-ui/FontIcon';
+import SwipeableViews from "react-swipeable-views";
```
Comment (arol) [inline imports/ui/Game.jsx]: JS! use single quotes

**Review by arol — CHANGES_REQUESTED** [review]

**Comment by jmporchet** [comment]
Aaand this is why you should not rely blindly on plugins to do your dirty work...

File: imports/ui/Game.jsx [hunk 2]
```diff
@@ -152,172 +162,194 @@ class Game extends React.Component {
...
     if (this.props.game.round > 2) {
-      return "END GAME";
+      return 'END GAME';
     }
-    return "NEXT ROUND";
+    return 'NEXT ROUND';
   }
 
   startPauselabel() {
     if (this.props.game.running) {
```
Comment (arol) [inline imports/ui/Game.jsx]: remove this if statement (it's doing nothing)

File: imports/ui/Game.jsx [hunk 3]
```diff
@@ -152,172 +162,194 @@ class Game extends React.Component {
...
+            labelColor='white'
+            style={{
+              margin: 'auto',
+              display: 'flex',
+              width: '90%',
+              height: 60,
+              color: 'blue'
+            }}
+            label={this.nextGameLabel()}
+            onTouchTap={() => this.nextRound()}
```
Comment (arol) [inline imports/ui/Game.jsx:229]: Just FYI, here, you're creating a new function on every `render` call and it's inefficient. You should pass it as `onTouchTap={this.nextRound}` and declaring the function as `nextRound = () => {…}`.

**Review by arol — CHANGES_REQUESTED** [review]

**Review by arol — DISMISSED** [review]

**Review by dbyington — APPROVED** [review]
I approve this pull request

**Pull request merged**